<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDoctor</class>
 <widget class="QDialog" name="DialogDoctor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>79</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>716</width>
    <height>79</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="nombre_doc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="espec">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Consultorio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="num_consul">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefono:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tel_doc">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="masinfo">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Más información</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
